docs/wired-client-architecture.md

```
@@ -0,0 +1,367 @@
+
+```bash
+# Glob pattern for multiple ports
+./bin/wired-client -port "/dev/ttyUSB*" -server http://192.168.1.100:8080
+```
+
+## Troubleshooting
+
+### Serial Communication Noise
+
+**Symptom:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)